### PR TITLE
master now requires ansible-2.6

### PR DESCRIPTION
**Pull request by richm**
test script configures ansible-2.6 repo

### CODE REVIEW

File: hack/get-machine-run-tests.sh [hunk 1]
```diff
@@ -268,6 +268,30 @@ EOF
 scp $runfile openshiftdevel:/tmp
 ssh -n openshiftdevel "bash $runfile"
 
+#      title: "enable ansible 2.6 repo"
+cat > $runfile <<EOF
+set -euxo pipefail
+pushd $OS_O_A_L_DIR > /dev/null
+curbranch=\$( git rev-parse --abbrev-ref HEAD )
+popd > /dev/null
+if [[ "\${curbranch}" == master ]] ; then
```
Comment (nhosoi) [inline hack/get-machine-run-tests.sh]: It'd be nice if you could allow ${curbranch} contains some keyword, e.g., "4.0" to execute the following if-clause in addition to just "master".  It'd help individual's debugging branch (which has to have the keyword in it, though :) to share the logic without modifying the script.

Comment (richm) [reply hack/get-machine-run-tests.sh]: yeah - somehow parse the branch into a number, and do `[ $branchnumber -gt 311 ]` - not sure how to do that - we have the same problem in aos-cd-jobs . . .

Comment (nhosoi) [reply hack/get-machine-run-tests.sh]: Ok.  It's not a big deal.  I'm setting /lgtm.

**Pull request merged**